Polygon_mesh_processing/examples/Polygon_mesh_processing/interpolated_corrected_curvatures.cpp

```diff
@@ -0,0 +1,55 @@
+#include <CGAL/Surface_mesh.h>
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <iostream>
```

Comment:
I do not know what  are the CGAL conventions, but I would put all the standard includes before the boost includes, and the boost includes  before the CGAL includes.

Reply:
What we do:  CGAL, then boost, then std. The rational being that we want to avoid compilation errors depending on the order of inclusion of CGAL headers. Doing this helps detecting such errors (even if we have a test compiling independently all headers isolated).

Reply:
As we cannot control in which way CGAL users will include headers, we want to test the worst possible scenario. CGAL headers are not allowed to introduce error in other headers, and we want to test the possibility as much as possible.

Reply:
Thanks for the feedback, I reordered the includes as described. CGAL -> Boost -> std.